spec/mailers/user_mailer_spec.rb

```diff
@@ -10,13 +10,29 @@
 
       it "sends the invitation" do
         expect(invite_email.to).to contain_exactly(user.email)
-        expect(invite_email.subject).to eq("You have been invited to #{organisation.name}")
+        expect(invite_email.subject).to eq("Invitation to join claims funding for mentor training")
```

Comment:
This is incorrect. "claims funding for mentor training" is not our official service name.